scripts/apps/ingest/views/settings/wufoo.html

```diff
@@ -0,0 +1,12 @@
+<div class="field">
+    <label for="wufoo_username" translate>Login</label>
+    <input type="text" id="wufoo_username" placeholder="{{:: 'Wufoo login'|translate }}"
+      ng-model="provider.config.wufoo_username" required
```

Comment:
Hi @gbbr thanks for the review.

I'm not sure about the indentation rules used in the client, but I've used the same as in other views (e.g. https://github.com/superdesk/superdesk-client-core/blob/master/scripts/apps/ingest/views/settings/emailConfig.html). How would you change that?